packages/cli/src/index.ts

```diff
@@ -30,15 +31,29 @@ program
 program
     .command("create")
     .description("Create a Semaphore project with a supported template.")
-    .argument("<project-directory>", "Directory of the project.")
+    .argument("[project-directory]", "Directory of the project.")
     // .option("-t, --template <template-name>", "Supported Semaphore template.", "hardhat")
     .action(async (projectDirectory) => {
+        let projectDir: string
+
+        if (!projectDirectory) {
+            const answers = await inquirer.prompt({
+                name: "project_name",
+                type: "input",
+                message: "Enter your project name:",
+                default: "my-app"
+            })
+            projectDir = answers.project_name
+        } else {
+            projectDir = projectDirectory
+        }
```

Comment:
```suggestion
        if (!projectDirectory) {
            const answers = await inquirer.prompt({
                name: "project_name",
                type: "input",
                message: "Enter your project name:",
                default: "my-app"
            })
            
            projectDirectory = answers.project_name
        }
```

`projectDirectory` can directly be used, since it's not a constant. But it's necessary to rename the `projectDir` variable below.